controllers/admin/AdminStatusesController.php

```diff
@@ -602,6 +602,19 @@ public function postProcess()
             if (!$this->access('add')) {
                 return;
             }
+            $langIds = Language::getIDs(false);
+
+            foreach ($langIds as $id_lang) {
+                $count = OrderState::countByLocalizedName(
+                    (string) $_POST['name_' . $id_lang],
```

Comment:
I think we're supposed to use `Tools::getValue()` but I'm no legacy expert 😅

Reply:
Fixed